cs-http-client/src/main/java/io/cloudslang/content/httpclient/actions/HttpClientAction.java

```diff
@@ -129,9 +129,10 @@
      *                                           The default will create no more than 2 concurrent connections per given route. Default value: 2
      * @param connectionsMaxTotal                The maximum limit of connections in total.
      *                                           The default will create no more than 2 concurrent connections in total. Default value: 20
```

Comment:
Why do we have changes for HttpClientAction in this PR?

Reply:
file was removed